omod/src/test/java/org/openmrs/module/idgen/web/search/LogEntrySearchHandlerTest.java

```diff
@@ -82,7 +82,65 @@ public void getSearchConfig_shouldReturnLogEntryByDateGenerated() throws Excepti
 	public void getSearchConfig_shouldReturnLogEntryByGeneratedBy() throws Exception {
 		MockHttpServletRequest req = request(RequestMethod.GET, getURI());
 		req.addParameter("generatedBy", USER_UUID);
+		req.addParameter("comment", "New");
+		SimpleObject result = deserialize(handle(req));
+		Assert.assertEquals(2, Util.getResultsSize(result));
+	}
+
+	@Test
+	public void getSearchConfig_shouldReturnLogEntryByGeneratedByAndComment() throws Exception {
+		MockHttpServletRequest req = request(RequestMethod.GET, getURI());
+		req.addParameter("generatedBy", USER_UUID);
+		req.addParameter("comment", "New");
+		SimpleObject result = deserialize(handle(req));
+		Assert.assertEquals(2, Util.getResultsSize(result));
+	}
+	@Test
+	public void getSearchConfig_shouldReturnLogEntryByGeneratedByAndCommentAndIdentifier() throws Exception {
+		MockHttpServletRequest req = request(RequestMethod.GET, getURI());
+		req.addParameter("generatedBy", USER_UUID);
+		req.addParameter("comment", "New");
+		req.addParameter("identifier", "100");
 		SimpleObject result = deserialize(handle(req));
 		Assert.assertEquals(2, Util.getResultsSize(result));
 	}
+
+	@Test
+	public void getSearchConfig_shouldReturnLogEntryBySourceAndComment() throws Exception {
+		MockHttpServletRequest req = request(RequestMethod.GET, getURI());
+		req.addParameter("source", LOG_ENTRY_SOURCE_UUID);
+		req.addParameter("comment", "New");
+		SimpleObject result = deserialize(handle(req));
+		Assert.assertNotNull(result);
+		Assert.assertEquals(2, Util.getResultsSize(result));
+	}
+		@Test
```

Comment:
Did you intentionally format this like so?

Reply:
That was an oversight on my side. Going to fix it now